### ISSUE TITLE
Special literal meaning "linebreak"

### ISSUE THREAD

**Issue by ger-benjamin** [geostyler/geostyler-style#548]
# Feature Request

Hello,

In some solutions, it's possible to have a "new line" between two texts. Example:
Wanted result: 
```
1234
My shape
```

With ArcGIS:
`"[FOKR]&vbNewLine& [FOKRNAME] "`

With GeoServer: 
```
            <Label>
              <ogc:Function name="Concatenate">
                <ogc:PropertyName>fokr</ogc:PropertyName>
                <ogc:Function name="Concatenate">
                  <ogc:Literal><![CDATA[
]]></ogc:Literal>
                  <ogc:PropertyName>fokrname</ogc:PropertyName>
                </ogc:Function>
              </ogc:Function>
            </Label>
```

**Describe the solution you'd like**
It would be great to have a special literal for that in the Geostyler style format. Maybe something like a WellKnownName ?
Or it is already existing and I missed it ?

**Comment by jansule** [comment]
Good point! This looks like something we do not support yet. Do you have a specific literal in mind?

Currently, we use `{{COLUMN_NAME}}` (wrapping the respective column name into two curly braces) for resolving placeholders. So this literal is already reserved.

@KaiVolland what do you think?

**Comment by ger-benjamin** [comment]
> Do you have a specific literal in mind?

That's the hardest part ! I think ``{{NEW_LINE}}`` or ``{{LINE_BREAK}}`` would be good.
Maybe there are other "WellKnownPlaceholder", we could add both as a new type ?

**Comment by jansule** [comment]
> That's the hardest part

True :grinning: 

My previous comment might have been too imprecise. Sorry for that. Using two curly braces is already reserved for resolving placeholders. `COLUMN_NAME` was only used to show that users can provide any column name there, e.g.

```
label: "count: {{COUNT}}"
```

would render a label as

```
count: 10
```

for a feature with the value `10` in a column `COUNT`.

So, using `{{NEW_LINE}}` or `{{LINE_BREAK}}` will not be an option, since users would not be able to reference columns with the names `NEW_LINE` or `LINE_BREAK`.

**Comment by KaiVolland** [comment]
Actually I'm not sure if and how should deal with the existing placeholder/template syntax as we now support functions in geostyler-style and especially the [use of feature properties](https://github.com/geostyler/geostyler-style/blob/24ea641b5392d037f9902ae5ce5ce9e8ad85ed77/functions.ts#L271).

The thing with the linebreak is indeed a problem that occured more often. E.g. here: https://github.com/geostyler/geostyler/issues/2008

My personal preference would be to make use of an existing simple concept like [escape sequences](https://learn.microsoft.com/en-us/cpp/c-language/escape-sequences?redirectedfrom=MSDN&view=msvc-170). So we could just use `\n` for a newlince. The parsers should then do the things they have to do to transform it to their syntax.

**Comment by jansule** [comment]
> So we could just use \n for a newlince

I think that's a good point. This also leads to users being able to just press `ENTER` in the input fields in the UI, which is already the case.

**Comment by ger-benjamin** [comment]
Okay, +1

**Comment by jansule** [comment]
Absolutely. I think we should also explicitly specify this in geostyler-style. Something like

```
type newlineToken = '/n';
```

**Comment by ger-benjamin** [comment]
Closed with #561
